frontend/src/components/ChargepointItem.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components/macro";
+
+export default function ChargepointItem({ chargepoint }) {
+  return (
+    <Wrapper>
+      <h2>{chargepoint.name}</h2>
+      <div>
```

Comment:
Maybe a `nav` and better open links a new tab and not direct users away from your app.